### PR TITLE
Character Creation Overhaul

### PR DESCRIPTION
**Pull request by Poojawa**
:cl: Poojawa
add: All markings, tails, ears, and snouts for Citadel races are color matrixed!
imageadd: all markings are on a per-limb basis, including Digitigrade legs!
imageadd: a bunch of tails were blessed with tail wagging sprites, Fish, Sharks, Fennecs, Wahs, raccoons, and others.
imageadd: Tiger markings + tail added, skunk tails improved via sprites from Virgo
tweak: tweaked some sprites to look better, but they absolutely could use a few extra passes for quality
add: HumanScissors in the Tools folder will permit anyone to contribute matrix'd markings to the sprite sheet, however Mam_markings is very, very full. Be careful.
tweak: Character preview was both optimized for taurs and bad-touched for better updating. I don't know if it'll be bad, but hey its better.
/:cl:

My magnum opis for SS13. Finally caving into the suggestions by Bhjin and going on a fucking spiritual quest to discover, code, sprite, and get the concept working completely. It's taken about 6 months I believe.

### What's new?

All Citadel anthro sprites are now converted to using a Color Matrix for the 3 color channels. All Sprites basically look like this now : 

And THIS is how it looks like in game

There's still some teething issues as per usual, but every limb should carry over its assigned marking and colors. Replaced limbs should carry over their markings as well, so you can, theoretically, mix and match body parts.

~~Tauric bodies don't have matrix yet, but I figured this is too important not to have test merged. ~~
Taurs have been converted, and tails redone for them. They're more or less the same as anthro tails, so tail waggle is possible

### Why should we care?

This system compresses down the icons drawn to just one per limb, it also enables digitigrade legs without the really weird ghost leg syndrome. This also fixes phantom sprites when missing limbs in general now so you'll know when you've lost a leg or arm.

### CODE REVIEW

**Comment by Poojawa** [comment]
>should should properly.

Should look. honk.

File: code/modules/mob/dead/new_player/sprite_accessories.dm [hunk 1]
```diff
@@ -1516,18 +1516,3 @@
...
-	hasinner = 1
-	extra = TRUE
-	extra_color_src = MUTCOLORS2
-	ckeys_allowed = list("invader4352")
-
-/datum/sprite_accessory/mam_tails/lunasune
-	name = "lunasune"
-	icon_state = "lunasune"
-	extra = TRUE
-	ckeys_allowed = list("invader4352")
```
Comment (deathride58) [inline code/modules/mob/dead/new_player/sprite_accessories.dm:1533]: ?

Comment (Poojawa) [reply code/modules/mob/dead/new_player/sprite_accessories.dm:1533]: extra stuff, it was already in the modular citadel file

**Comment by Poojawa** [comment]
fuck you icondiffbot

**Review by deathride58 — APPROVED** [review]
This is a big huge PR that will probably cause conflicts with other PRs. Can be merged on review from any other maintainer

**Comment by silicons** [comment]
image
nekos are still broken.

**Pull request merged**